### ISSUE TITLE
Expired message display handling

### ISSUE THREAD

**Issue by fentie** [18F/dolores-landingham-bot#161]
I was thinking about the concept of messages whose `end_date` has passed... it makes sense to me to supply some sort of visual affordance to the admins, if not hiding them all together. Perhaps some sort of shading across those records in the index view?

**Comment by jessieay** [comment]
I could see us having a daily cron job that soft deletes expired scheduled messages. Is that kind of what you were thinking? Agreed that we don't want admins to have to sift through old msgs all the time.

**Comment by fentie** [comment]
It'd depend on how message expiry was used, really... I could see someone wanting to make "seasonal" messages that came up on a recurring basis but were not always active, in which case they would edit the `end_date` every once in awhile vs. recreating the message each time. Alternatively we could add additional sorting to drop them to the bottom of the list, or add a UI option to toggle showing expired messages or not. There are a lot of patterns out there to choose from. 

It might be best to just let the system run with them available for awhile and see how the users employ them?

**Comment by jessieay** [comment]
@fentie I agree with your "just let the system run with them available for awhile and see how the users employ them?" idea. Will keep this issue open and see if we get any suggestions for hiding expired messages.

Seems like seasonal messages are separate from expired / expiring messages. Another interesting idea although not one I've hard suggested by users yet.
